redwood-treehouse-host/src/commonMain/kotlin/app/cash/redwood/treehouse/TreehouseAppContent.kt

```diff
@@ -74,9 +73,9 @@ internal class TreehouseAppContent<A : AppService>(
   private val dispatchers: TreehouseDispatchers,
   private val appScope: CoroutineScope,
   private val eventPublisher: EventPublisher,
-  private val source: TreehouseContentSource<A>,
   private val codeListener: CodeListener,
-) : Content {
+  private val source: TreehouseContentSource<A>,
```

Comment:
[nitpick] Was the reordering deliberate?

Reply:
Yep! As a rule I try to order variables by their scope; most-singleton things first followed by least-singleton things. In this case the codeListener has a longer lifetime than the source.

(I don’t impose this on other people)